generate/funcs.go

```diff
@@ -142,7 +142,14 @@ func GenerateFunctions(from map[string][]parser.Builtin, categorize bool) []byte
 				continue
 			}
 			args := fn.Types.String()
-			ret := fn.ReturnType.String()
+			retType := fn.ReturnType(nil)
+			// TODO(bdarnell): ReturnType(nil) returns nil for some functions.
+			// Empirically, these should be "anyelement", but is there a better way
+			// to handle this?
+			ret := "anyelement"
```

Comment:
We have a method called `FixedReturnType()` on a `parser.Builtin` that does exactly what you're doing here, so this should become `fn.FixedReturnType().String()`.